### ISSUE TITLE
natural=mud doesn't render in rivers

### ISSUE THREAD

**Issue by chillly** [gravitystorm/openstreetmap-carto#115]
natural=mud now renders around the coastline, but not in rivers. The example below shows the join between coast and river and the cutoff is obvious.

Example: http://www.openstreetmap.org/#map=16/53.6995/-0.4381

**Comment by malcolmh** [comment]
If the waterway=riverbank tag is changed to natural=water, then the rendering is correct. These two tags ought to produce the same result, but don't.

**Comment by imagico** [comment]
I see no indication of areas tagged natural=water and waterway=riverbank being treated any differently - if they are please open a new issue and show an example.  Note any special issues with natural=mud will be solved by #1497.

This problem here (different handling of coastline and water areas in relation to landcovers) is still unsolved - it will require use of water polygons for the ocean rendering.

**Comment by malcolmh** [comment]
Look here: http://www.openstreetmap.org/#map=14/53.7058/-0.5772. I changed one of the waterway=riverbank polygons to natural=water and the mud re-appeared.

**Comment by imagico** [comment]
#1497 is not yet rolled out - if there was any difference before it should be gone after this change.

By the way - if you change waterway=riverbank to natural=water please always add water=river.

**Comment by malcolmh** [comment]
OK, I will wait until roll-out to see if this issue is then cleared. Thanks!

**Comment by math1985** [comment]
Closing this now, please let us know if there is still a problem after #1497 is rolled out.
